internal/policygen/policygen.go

```diff
@@ -43,7 +43,6 @@ type RunArgs struct {
 	OutputPath string
 }
 
-// Run executes main policygen logic.
```

Comment:
This is a public func. It doesn't need a comment?

Reply:
it needs. you caught this right in the meantime of me testing the CI should fail this case.

Reply:
It's reverted in the next commit.